mne/report.py

```diff
@@ -174,6 +174,16 @@ def _is_bad_fname(fname):
         return ''
 
 
+def _get_fname(fname):
+    """Get fname without -#-"""
+    if '-#-' in fname:
+        fname = fname.split('-#-')[0]
+    else:
+        fname = op.split(fname)[1]
```

Comment:
It's the same according to the documentation here: https://docs.python.org/2/library/os.path.html. I tried them for some filenames with spaces and it seemed to give the same result. I changed it anyway :)